packages/flame/test/_resources/test_text_file.txt

```diff
@@ -0,0 +1 @@
+This is sample text file for AssetsCache Unit testing.
```

Comment:
Should this file be under the _resources folder as well?

Reply:
yes. If you want to make another folder then I can change.